Mage.Sets/src/mage/cards/n/Nethergoyf.java

```diff
@@ -0,0 +1,128 @@
+package mage.cards.n;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.costs.Cost;
+import mage.abilities.costs.CostsImpl;
+import mage.abilities.costs.common.ExileFromGraveCost;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.dynamicvalue.common.CardTypesInGraveyardCount;
+import mage.abilities.effects.common.continuous.SetBasePowerToughnessPlusOneSourceEffect;
+import mage.abilities.hint.HintUtils;
+import mage.abilities.keyword.EscapeAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.SubType;
+import mage.constants.Zone;
+import mage.filter.FilterCard;
+import mage.filter.predicate.mageobject.AnotherPredicate;
+import mage.game.Game;
+import mage.target.common.TargetCardInYourGraveyard;
+import mage.util.CardUtil;
+
+import java.awt.*;
+import java.util.Collection;
+import java.util.Objects;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+/**
+ * @author Susucr
+ */
+public final class Nethergoyf extends CardImpl {
+
+    private static final DynamicValue powerValue = CardTypesInGraveyardCount.YOU;
```

Comment:
this can just be inlined right?

Reply:
All the Tarmogoyf-like have their DynamicValue stored as private final. Just following the pattern there.
But indeed, could be just inlined.

Reply:
Ok there isn't that many, just Tarmogoyf, Consuming Blob and its token.
I don't mind this style though.